dbt/include/snowflake/macros/materializations/incremental.sql

```diff
@@ -11,20 +10,41 @@
        for (presumably) faster overall incremental processing.
 
        Low-level specifics:
+       If an invalid option is specified, then we will raise an
+       excpetion with corresponding message.
+
        Languages other than SQL (like Python) will use a temporary table.
        With the default strategy of merge, the user may choose between a temporary
        table and view (defaulting to view).
-       The append strategy can use a view because it will run a single INSERT statement.
-       When the unique_key is none,
-       then we can use a view because it will run a single INSERT statement.
-       Otherwise, play it safe by using a temporary table.
```

Comment:
Why was the "play it safe comment removed?

Reply:
in the high level principles it looks as though we go into detail as to why we would use the temporary table in regards to the `delete+insert` option.

Reply:
I'm not against completely re-organizing it per se. But I also don't see a need to re-org it dramatically. My default approach is to change as little as necessary.

I'd vote for adding this back:
```
 Otherwise, play it safe by using a temporary table.
 ```